README.md

```diff
@@ -47,6 +47,76 @@ Click **Add user** and fill in the required details, and check the “**Allow AP
 
 For more instructions, check out examples folder.
 
+## A usage example for creating a server
+
+Below is an example configuration on how to create a server using the Terraform provider. Note that currently, only 10 concurrent servers creation is possible per account. For more examples, visit [official Terraform documentation](https://registry.terraform.io/providers/UpCloudLtd/upcloud/latest/docs).
```

Comment:
Repeats what we already know at the beginning of the sentence. Though if you decide to shorten the heading, maybe this is ok.